package.json

```diff
@@ -39,24 +41,38 @@
     "@babel/preset-env": "^7.3.4",
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.1",
+    "babel-jest": "^24.5.0",
     "babel-loader": "^8.0.5",
     "css-loader": "^2.1.1",
+    "cypress": "^3.2.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.11.2",
     "eslint": "^5.3.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-react": "^7.12.4",
     "file-loader": "^3.0.1",
     "html-webpack-plugin": "^3.2.0",
-    "jest": "^24.4.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^24.5.0",
     "node-sass": "^4.11.0",
+    "react-test-renderer": "^16.8.6",
```

Comment:
`react-test-renderer` is not used, so it should be deleted from the dependencies list.